### ISSUE TITLE
How to build multilayer interaction network through ILSM?

### ISSUE THREAD

**Issue by ZhangJingyiiii** [WeichengSun/ILSM#2]
I have successfully used the example data for analysis. So how can i use my own data to analysis, or depend on other package?

**Comment by WeichengSun** [comment]
Hi @ZhangJingyiiii, I'm glad you're interested in our project.
So if you're still wondering, hopefully the following will help:
1. There is no need to rely on other R packages for the data to be analysis.You only need to provide the multilayer network data according to our specifications.
2. We are keen that you can use a set of matrix pairs, each matrix represents an interaction layer, and the rows of the two matrices must correspond to the interconnection nodes.
3. You can also use the function [igraph_from_matrices](https://github.com/WeichengSun/ILSM/blob/main/R/igraph_from_matrices.R) to transform the over matrix pairs into "igraph" data that you can use and visualize. Or use the function [build_net](https://github.com/WeichengSun/ILSM/blob/main/R/build_net.R) for exercises with random generative networks.

**Comment by ZhangJingyiiii** [comment]
Thank you very much for your help. I have successfully run my data.

**Comment by ZhangJingyiiii** [comment]
I have one more question: Is it possible to calculate different interconnection motifs separately? My network is too large to compute all motifs, so I would greatly appreciate your guidance on this matter.

**Comment by WeichengSun** [comment]
Indeed, we learned that for large networks, the algorithm would be too slow because of the R logic, but we would add C++ later.

At present, for your needs, it is necessary to consider adding a parameter (ic_number) to the functions (icmotif_count and  icmotif_role) to calculate the top 36 motifs and top 50 roles with a max number of interconnection code  <= 2. All you need to do is add ic_number=2

**Comment by yanriver** [comment]
An update of this package has provided examples.
